app/models/webhook_header.rb

```diff
@@ -0,0 +1,22 @@
+# == Schema Information
+#
+# Table name: webhook_headers
+#
+#  id         :integer          not null, primary key
+#  webhook_id :integer
+#  name       :string(255)
+#  value      :string(255)
```

Comment:
> Right now we want to focus on notifications about new repositories/tags being created. In the near future we will support about notifications for delete events

I'd hope _all_ registry's notifications referring to some repo in the namespace get forwared. Why filter (see you own comment)?

> What kind of other notifications would you like to receive and why?

There could be notifications e.g. about failed logins, new registrations etc. But as I sad personally I don't have a specific need.

> We decided to leave filtering to the final endpoint.

I'm fine with this design decision. I'd suggest to state this scope explicitely in the opening comment of #794.